### PR TITLE
Add org.gnome.baobab

### PR DESCRIPTION
**Pull request by A6GibKm**
Continuation of #1573.

### CODE REVIEW

File: org.gnome.baobab.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    "sdk" : "org.gnome.Sdk",
+    "command" : "baobab",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=host:ro",
+        "--filesystem=~/.var/app:ro",
+        "--talk-name=org.gtk.vfs.*",
+        "--system-talk-name=org.freedesktop.hostname1",
```
Comment (barthalion) [inline org.gnome.baobab.json:14]: What is it for, displaying local hostname?

Comment (A6GibKm) [reply org.gnome.baobab.json:14]: I do not know...  As far as I know it was in the previous pr and upstream.

Comment (barthalion) [reply org.gnome.baobab.json:14]: Seems so, without it, it just says "Computer". I'd drop it, it looks very uncommon to request it.

Comment (A6GibKm) [reply org.gnome.baobab.json:14]: ...but the immersion :disappointed:

**Comment by A6GibKm** [comment]
It seems the app does not work in x11, even with the `--socket=x11` flag.

File: org.gnome.baobab.json (hunk 1 above)
Comment (A6GibKm) [reply org.gnome.baobab.json:14]: I am not sure how I feel about 'just because it is uncommon'.

Comment (A6GibKm) [reply org.gnome.baobab.json:14]: Ok, so flatpak is not able to see `\` anyways. I am dropping this.

**Comment by barthalion** [comment]
It works absolutely fine for me on X11.

**Comment by A6GibKm** [comment]
Ok, might been an issue on my side. I will test later using X11.

**Comment by barthalion** [comment]
bot, build org.gnome.baobab

**Comment by nedrichards** [comment]
Yeah, this was working fine for me in x11. I'm still not entirely clear who needs to be added to this when merged? Other than that lgtm (helped me clean up 2gb of storage yesterday!)

**Pull request closed without merging**